compatibility/modules/cxf/src/test/resources/http-security-conf-flow-httpn.xml

```diff
@@ -39,7 +39,7 @@
     
     <flow name="secure">
         <httpn:listener path="/services/Echo" config-ref="listenerConfig" allowedMethods="POST" />
-        <httpn:basic-security-filter realm="mule-realm"/>
+        <httpn:basic-security-filter realm="mule-realm" attributes="#[message.attributes]"/>
```

Comment:
can we infer the attributes? as this is here, it will be very disruptive for the migration